examples/RFToBModeGeneration.ipynb

```diff
@@ -151,109 +60,133 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# BMode Image (expected)\n",
```

Comment:
MouseLiverB in https://data.kitware.com/#collection/57b5c9e58d777f126827f5a1/folder/607efe062fa25629b9f62c18 was created by cropping larger versions of RF and B mode in roughly corresponding position, not directly from the RF version there.

Reply:
Could you provide more background on the origins of this data? Was `itk::BModeImageFilter` used to compose the B-mode image in the first place or was a different workflow used?

Reply:
Either WinProbe's RF->BMode conversion was used or [this](https://github.com/PlusToolkit/PlusLib/blob/e2d1a2b9056d779338771fb6c5df8884fadad319/src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx#L296-L333), not `itk::BModeImageFilter`. The original consisted of 3 parallel, partly overlapping sweeps. Total 1080 frames. Here I cropped only 3 frames somewhere in liver.

Reply:
Thanks for the explanation. I'll go back through the tests in ITKUltrasound and see if there's a workflow I can reproduce more closely in the notebook.

Reply:
[carotid_longitudinal.mha](https://data.kitware.com/#item/5cfe6e068d777f072b1fa6c9) and [carotid_transverse.mha](https://data.kitware.com/#item/5cfe6e318d777f072b1fa72e) are both potential RF candidates in the [ITKUltrasound data folder](https://data.kitware.com/#folder/58ee3b438d777f16d095fd83) but neither are currently used in testing and do not have BMode counterparts for comparison. Other existing input files are 2D and do not appear to contain the spectral content we'd ideally discuss in this example.

@dzenanz @thewtex  Is there anywhere else I could find publicly available RF data that might have a good baseline BMode comparison? Or are there brightness transforms I can apply to the output of `itk.b_mode_image_filter` when run on the `MouseLiverRF.mha` input so that the output better matches `MouseLiverB.mha`?

For comparison, here is the output from `itk.b_mode_image_filter` vs `MouseLiverB.mha`:

![compare-bmode](https://user-images.githubusercontent.com/40648863/116148646-2a0e4d00-a6af-11eb-94d1-b592b0b53a46.png)

Reply:
You could pick one of the recent phantom cases? e.g. this one: https://drive.google.com/drive/u/1/folders/16foHYzFtfyhCykJuODQ6SF67AponTK1H
Is it a problem that these scans are mostly of a homogenous part of a CIRS phantom?

Reply:
Can we use that data in a public repo? @thewtex might be able to better answer how homogeneity impacts the performance of this filter..

Reply:
Homogeneity is not a problem for the filter. The question is whether the resulting plots will be meaningful, from human interpretation and learning point of view.